core/genesis.go

```diff
@@ -549,8 +579,17 @@ func (g *Genesis) toBlockWithRoot(root common.Hash) *types.Block {
 			head.RequestsHash = &types.EmptyRequestsHash
 			requests = make(types.Requests, 0)
 		}
+		// If Isthmus is active at genesis, set the WithdrawalRoot to the storage root of the L2ToL1MessagePasser contract.
+		if g.IsIsthmus() {
+			if storageRootMessagePasser == (common.Hash{}) {
```

Comment:
Will it be `common.Hash{}` or [types.EmptyRootHash](https://github.com/ethereum/go-ethereum/blob/aa636921291744af082cefbe4dd811fb58fbde63/core/types/hashes.go#L27) at this abstraction?

Reply:
My understanding is that if there is no contract deployment, this woulld be an empty hash and not a empty root hash?

Reply:
+1, if the contract is absent then I believe the storage-root is zeroed.